Documentation/Evolution/CharacterClasses.md

```diff
@@ -0,0 +1,354 @@
+# Regular Expression Literals: Character Classes
```

Comment:
I wonder if the title should be less regex specific, e.g. "Declarative String Processing Character Classes"